src/FastRouteRouter.php

```diff
@@ -326,6 +381,80 @@ private function injectRoute(Route $route)
         }
 
         $this->router->addRoute($methods, $route->getPath(), $route->getPath());
-        $this->routes[$route->getName()] = $route;
+    }
+
+    /**
+     * Get the dispatch data either from cache or freshly generated by the
+     * FastRoute data generator. If caching is enabled store the freshly
+     * generated data to file.
+     *
+     * @return array
+     */
+    private function getDispatchData()
+    {
+        if ($this->hasCache) {
+            return $this->dispatchData;
+        }
+
+        $dispatchData = $this->router->getData();
+
+        if ($this->cacheEnabled) {
+            $this->cacheDispatchData($dispatchData);
+        }
+
+        return $dispatchData;
+    }
+
+    /**
+     * Load dispatchData from cache
+     *
+     * @return void
+     * @throws Exception\RuntimeException If the cache file contains invalid data
+     */
+    private function loadDispatchData()
+    {
+        if (! is_file($this->cacheFile)) {
```

Comment:
can `is_file` be skipped, and instead use something like a direct `include` statement? That would prevent a stat call here